### PR TITLE
Ft app  structure

### PR DESCRIPTION
**Pull request by DjibrilM**
## In the following pull request I 
- implemented the application structure 🔄
- integrated eslint and stylelint🚥
- integrated tailwind CSS(for easy styling)   🌈
- introduced application documentation(README.md)📕

### UI screenshot 
Screenshot_20230722_050232

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+
+function App() {
+  return (
+    <main className='w-full h-screen bg-slate-200'>
+      <Header />
+      <Routes>
+        <Route element={<Books />} path='/' />
+      </Routes>
+    </main>
```
Comment (usorfaitheloho) [inline src/App.jsx]: Kindly note that in the project requirement, you were asked to add two routes, for this two components 

Project requirement :

image

File: src/components/Book.jsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <div className="flex items-center w-1/5 h-full  justify-end   pl-7">
+                <button className='px-10 py-4 active:bg-red-300 active:text-red-600 mt-3 text-red-500 rounded-md bg-red-200 border-2 border-red-300'>
+                    Delete
+                </button>
+            </div>
+        </li>
+    )
+}
+
+export default Book
```
Comment (usorfaitheloho) [inline src/components/Book.jsx:21]: you did well with your `books` component, kindly note that you are to have the categories components just as you should have the routes too 😄 , to enable a dynamic content creation of your page

File: src/screens/Books.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        <>
+            <ul className='max-w-[1300px] pr-10 h-[calc(100vh-250px)] books-container flex flex-col gap-4  overflow-auto  border m-auto mt-10'>
+                <Book />
+            </ul>
+            <Form />
+        </>
+    )
+}
+
+export default Books
```
Comment (usorfaitheloho) [inline src/screens/Books.jsx:36]: please note that your `Book ` component should take the book data as a prop and renders the title and the author, so your book component should have title and author as props 😄

**Review by usorfaitheloho — CHANGES_REQUESTED** [review]
> Hi @DjibrilM  ,
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

  [ ](https://camo.githubusercontent.com/679757134a8fee76cfe1941f9f3ecb0e0cf56dd02da4498268ce31f0dacdfca3/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)
> 
> ## Highlights
> * Good UI✔️
> * Descriptive pull request ✔️
> * Used GitHub flow ✔️
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.jsx (hunk 1 above)
Comment (rloterh) [reply src/App.jsx]: well resolved! ✔️

File: src/screens/Books.jsx (hunk 3 above)
Comment (rloterh) [reply src/screens/Books.jsx:36]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Djibril,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on implementing recommended changes!

Cheers..Happy coding! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**